clients/client-python/setup.py

```diff
@@ -6,16 +6,33 @@
 from setuptools import find_packages, setup
 
 
+try:
+    with open("README.md") as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = "No README.md file found"
+
 setup(
     name="gravitino",
     description="Python lib/client for Gravitino",
-    version="0.5.0",
-    long_description=open("README.md").read(),
+    version="0.5.0.dev12",
+    long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/datastrato/gravitino",
     author="datastrato",
+    author_email="support@datastrato.com",
     python_requires=">=3.8",
-    packages=find_packages(include=["gravitino", ".*"]),
+    packages=find_packages(exclude=["tests*"]),
+    classifiers=[
+        'Development Status :: 3 - Alpha',
```

Comment:
What is the meaning of `3 - Alpha`, does it have special purpose?

Reply:
This is Pypi.org's development status classify, https://pypi.org/search/?c=Development+Status+%3A%3A+3+-+Alpha